src/app/tests/TestAclAttribute.cpp

```diff
@@ -121,29 +109,63 @@ class MockInteractionModelApp : public chip::app::ReadClient::Callback
 namespace chip {
 namespace app {
 
-class TestAclAttribute
+class TestAclAttribute : public ::testing::Test
 {
 public:
-    static void TestACLDeniedAttribute(nlTestSuite * apSuite, void * apContext);
+    static void SetUpTestSuite()
+    {
+
+        mpTestContext = new chip::Test::AppContext;
+
+        mpTestContext->SetUpTestSuite();
+    }
+    static void TearDownTestSuite()
+    {
+        mpTestContext->TearDownTestSuite();
+        if (mpTestContext != nullptr)
```

Comment:
you are using mpTestContext above ... so it is not nullptr or it would segfault.

Should we move the tear down here or just not have the check and trust that teardown follows setup?